src/lib/security/selinux_config.rb

```diff
@@ -25,92 +25,156 @@ module Security
   class SelinuxConfig
     include Yast::Logger
     Yast.import "Bootloader"
+    Yast.import "ProductFeatures"
 
     attr_reader :policy
 
+    # Constructor
     def initialize
-      propose_policy if Yast::Mode.installation
-      @policy = configured_policy
+      @policy = initial_policy
+    end
+
+    # Return the initial SELinux policy
+    #
+    # It can be the proposed one if running in an installation or the configured policy for a
+    # running system
+    #
+    # @return [Symbol]
+    def initial_policy
+      @initial_policy ||=
+        begin
+          propose_policy if Yast::Mode.installation
+          configured_policy
+        end
     end
 
     # Returns the policy applied in the running system
+    #
+    # @note the system can be booted with a different SELinux policy that the configured one. To
+    #  know the running policy getenforce tool is used.
+    #
+    # @return [Symbol] running SELinux policy if command executed successfully; :disabled otherwise
     def running_policy
       Yast::Execute.locally!(GETENFORCE_PATH, stdout: :capture).chomp.downcase.to_sym
     rescue Cheetah::ExecutionFailed
       log.debug("`#{GETENFORCE_PATH}` not available. SELinux or selinux-tools missing")
       :disabled
     end
 
-    # If current policy value is different to the configured one
-    def changed?
-      @policy != configured_policy
+    # Set the policy to given value
+    #
+    # @note using nil means to set SELinux policy as disabled.
```

Comment:
why the nil case is needed?

Reply:
Actually not needed, it is just a _facility_ for disabling it. We can remove it.